docs/netlify.toml

```diff
@@ -470,6 +470,11 @@
   to = "/reference/php"
   status = 301
 
+[[redirects]]
+  from = "/integrations/java"
+  to = "/api/sdk"
```

Comment:
@jasonmccallister I will update this to point to the Java SDK autogen'd reference once it is ready.